### ISSUE TITLE
css order is wrong 

### ISSUE THREAD

**Issue by hpugongying** [webpack/webpack#6747]
**Do you want to request a *feature* or report a *bug*?**
report a bug

**What is the current behavior?**
CSS compilation results should be in order of require. 

**If the current behavior is a bug, please provide the steps to reproduce.**

// common.scss
`html {
    color: purple;
}`

// index.scss 
`html {
    color: blue;
}`

index.js 
`require('./common.scss');
 require('./index.scss');
`

//result 
html  color is purple

**What is the expected behavior?**
html  color is blue

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as the browser version, Node.js version, webpack version, and Operating System.**
node v8.9.4
webpack: 4.1.0
browser:  chrome 64.0.3282.186
os: mac os 10.13.2

**Comment by Legends** [comment]
Yes, they should be in order as they are `required`. How does your webpack.config.js for `scss` files look like?

**Comment by sokra** [comment]
Thanks for your report. It would be great if you reduce your issue to a small reproducible example. Best put this example into a github repository together with instructions how to get to the problem.
